prod_cockpit_demo/domd.xml

```diff
@@ -12,7 +12,7 @@
   <remote name="epam" fetch="ssh://git@gitpct.epam.com" />
 
   <!-- defaults -->
-  <default remote="agl" sync-j="4" revision="refs/tags/icefish/9.0.2"/>
+  <default remote="agl" sync-j="4" revision="refs/tags/koi/11.0.1"/>
```

Comment:
Pay attention, that this change is commented out in following commit "DomD: change to dunfell".
https://github.com/xen-troops/manifests/pull/88/commits/b3474cbd5b2507f3f14df4e18aff78742562fc98#diff-8633a38636ea7c613baeffead3c390ea6eaca7d41c6f67d843d1e742d2811629R16

If it is intentionally then I'm OK.